lib/preboots/feedsme.js

```diff
@@ -15,5 +15,11 @@ module.exports = function feedme(app, options, callback) {
     uri: uri
   });
 
+  app.feedsmeV1 = new Feedsme({
```

Comment:
Without this (since the APIs are interdependent) calling v1 `carpenterd` would result in `feedsme` v2 being called, which would then call `carpenterd` v2 for dependent builds.

Reply:
Is that a problem? shouldn't they just start talking to the v2 endpoints? and the v1 are just there as shims to maintain some sense of compatibility from the warehouse.ai service?